rs/nns/governance/canister/governance.did

```diff
@@ -577,6 +626,62 @@ type Neuron = record {
   transfer : opt NeuronStakeTransfer;
   known_neuron_data : opt KnownNeuronData;
   spawn_at_timestamp_seconds : opt nat64;
+
+  // When a neuron is first created, this is set to create_timestamp_seconds.
+  //
+  // This is part of the "periodic confirmation" feature, which was proposed and
+  // (overwhelmingly) adopted in NNS motion proposal 132411:
+  // https://dashboard.internetcomputer.org/proposal/132411
```

Comment:
```suggestion
```

I would focus here on what is the case in the code and less on how we got to this decision

Reply:
The proposal contains an explanation of the behavior. I think it's better to link out than to repeat the explanation inline here, because it is long, and we would need to repeat it at many places.

Reply:
I think for sharing this with the community I agree. 
For the final version that we actually merge to the code, it makes more sense not to having to refer to external resources. But I let the final reviewer of the PR decide

Reply:
We can maybe move this further down, and make it **not** required reading.